README.md

```diff
@@ -1,10 +1,47 @@
 # npm-release-management-orb
 [![CircleCI Build Status](https://circleci.com/gh/forcedotcom/npm-release-management-orb.svg?style=shield&circle-token=7a6dcc6c02f82515aec6533ed1c7253ef38e6e13 "CircleCI Build Status")](https://circleci.com/gh/forcedotcom/npm-release-management-orb) [![CircleCI Orb Version](https://img.shields.io/badge/endpoint.svg?url=https://badges.circleci.io/orb/salesforce/npm-release-management)](https://circleci.com/orbs/registry/orb/salesforce/npm-release-management) [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://raw.githubusercontent.com/forcedotcom/npm-release-management-orb/develop/LICENSE.txt)
 
+- [npm-release-management-orb](#npm-release-management-orb)
+  - [Usage](#usage)
+  - [Environment Variable Reference](#environment-variable-reference)
+  - [How To Contribute](#how-to-contribute)
+
 ## Usage
 
 Example use-cases are provided on the orb [registry page](https://circleci.com/orbs/registry/orb/salesforce/npm-release-management#usage-examples). Source for these examples can be found within the `src/examples` directory.
 
+This orb is designed to make releasing npm packages straightforward and with as little configuration as possible. To get started you will need to:
+
+1. Add the orb to your config.yml
+
+    ```
+    orbs:
+      release-management: salesforce/npm-release-management@x.y.z
+    ```
+
+    NOTE: We recommend pinning the major version (e.g. `salesforce/npm-release-management@1`) so that you automatically recieve all minor and patch updates.
+
+2. Setup the required environment variables.
+
+    Please see the [Environment Variable Reference](#environment-variable-reference) for the required environment variables.
+
+3. Add a user key to your SSH Keys in CircleCi
+
+    As part of the release job we create a new git tag based on the version in the package.json. This requires that you add a user key that has the permissions to push to github. To do this, simply go to the `SSH Keys` tab in the project settings and follow the directions for adding user keys.
+
+4. Follow the [examples] (https://circleci.com/orbs/registry/orb/salesforce/npm-release-management#usage-examples) for how to setup your config.yml
+
+## Environment Variable Reference
+
+| Variable                           | Description                                                        | Required |
+|------------------------------------|--------------------------------------------------------------------|----------|
+| AWS_ACCESS_KEY_ID                  | access key id for aws s3                                           | Yes      |
```

Comment:
Is it worth noting that the AWS* and SALESFORCE_KEY is only required for the release-signed-package?